Mathlib/SetTheory/Game/Ordinal.lean

```diff
@@ -155,29 +155,55 @@ noncomputable def toPGameEmbedding : Ordinal.{u} ↪o PGame.{u} where
   map_rel_iff' := @toPGame_le_iff
 
 /-- The sum of ordinals as games corresponds to natural addition of ordinals. -/
-theorem toPGame_add : ∀ a b : Ordinal.{u}, a.toPGame + b.toPGame ≈ (a ♯ b).toPGame
+theorem toPGame_add : ∀ a b : Ordinal.{u}, (a ♯ b).toPGame ≈ a.toPGame + b.toPGame
   | a, b => by
     refine ⟨le_of_forall_lf (fun i => ?_) isEmptyElim, le_of_forall_lf (fun i => ?_) isEmptyElim⟩
-    · apply leftMoves_add_cases i <;>
-      intro i <;>
-      let wf := toLeftMovesToPGame_symm_lt i <;>
-      (try rw [add_moveLeft_inl]) <;>
-      (try rw [add_moveLeft_inr]) <;>
-      rw [toPGame_moveLeft', lf_congr_left (toPGame_add _ _), toPGame_lf_iff]
-      · exact nadd_lt_nadd_right wf _
-      · exact nadd_lt_nadd_left wf _
     · rw [toPGame_moveLeft']
       rcases lt_nadd_iff.1 (toLeftMovesToPGame_symm_lt i) with (⟨c, hc, hc'⟩ | ⟨c, hc, hc'⟩) <;>
-      rw [← toPGame_le_iff, ← le_congr_right (toPGame_add _ _)] at hc' <;>
+      rw [← toPGame_le_iff, le_congr_right (toPGame_add _ _)] at hc' <;>
       apply lf_of_le_of_lf hc'
       · apply add_lf_add_right
         rwa [toPGame_lf_iff]
       · apply add_lf_add_left
         rwa [toPGame_lf_iff]
+    · apply leftMoves_add_cases i <;>
+      intro i <;>
+      let wf := toLeftMovesToPGame_symm_lt i <;>
+      (try rw [add_moveLeft_inl]) <;>
+      (try rw [add_moveLeft_inr]) <;>
+      rw [toPGame_moveLeft', ← lf_congr_left (toPGame_add _ _), toPGame_lf_iff]
+      · exact nadd_lt_nadd_right wf _
+      · exact nadd_lt_nadd_left wf _
 termination_by a b => (a, b)
 
-@[simp]
-theorem toPGame_add_mk' (a b : Ordinal) : (⟦a.toPGame⟧ + ⟦b.toPGame⟧ : Game) = ⟦(a ♯ b).toPGame⟧ :=
+theorem toPGame_add_mk' (a b : Ordinal) : (⟦(a ♯ b).toPGame⟧ : Game) = ⟦a.toPGame⟧ + ⟦b.toPGame⟧ :=
   Quot.sound (toPGame_add a b)
 
+/-- The product of ordinals as games corresponds to natural multiplication of ordinals. -/
+theorem toPGame_mul : ∀ a b : Ordinal.{u}, (a ⨳ b).toPGame ≈ a.toPGame * b.toPGame
+  | a, b => by
```

Comment:
```suggestion
theorem toPGame_nmul (a b : Ordinal.{u}) (a ⨳ b).toPGame ≈ a.toPGame * b.toPGame := by
```

Reply:
Putting `a` and `b` before the colon doesn't work, since it breaks the `termination_by` syntax.

Reply:
Ugh, annoying. @nomeata? 😇

Reply:
Stuff before the colon is in scope for `termination_by`, so just don't bind it there again?